### PR TITLE
set node-version output (2022 edition)

### PR DESCRIPTION
**Pull request by panva**
_I am unable to update the original PR with a rebase and re-build so I'm opening a new one._

**Description:**
This adds a `node-version` output similar to [`setup-python`](https://github.com/actions/setup-python/blob/main/action.yml#L23-L25) so that the exact installed node version can be used e.g. as part of a cache key for following build steps.

**Related issue:**

#150 #173

**Check list:**
- [ ] Mark if documentation changes are required.
- [ ] Mark if tests were added or updated to cover the changes.

### CODE REVIEW

**Comment by dmitry-shibanov** [comment]
Hello @panva. Could you please run `npm run build` and `npm run format` commands?

**Comment by panva** [comment]
> Could you please run npm run build and npm run format commands?

done.

File: src/main.ts [hunk 1]
```diff
@@ -39,6 +40,17 @@ export async function run() {
...
 
+    // Output version of node is being used
+    let installedVersion = '';
+    await exec.exec('node', ['--version'], {
+      listeners: {
+        stdout: data => {
+          installedVersion += data.toString();
+        }
+      }
+    });
```
[the remark this answers is not in the document]
Comment (panva) [reply src/main.ts]: done.

**Comment by dmitry-shibanov** [comment]
Could you please revert changes to previous one and add mock for `getExecOutput` in `installer.test.ts` ?
```
    // @actions/exec
    getExecOutputSpy = jest.spyOn(exec, "getExecOutput");
    getExecOutputSpy.mockImplementation(() => "16.15.0");
```

**Review by dmitry-shibanov — APPROVED** [review]

**Review by e-korolevskii — APPROVED** [review]

**Review by marko-zivic-93 — APPROVED** [review]

**Comment by panva** [comment]
Thank you guiding this through @dmitry-shibanov.

**Pull request merged**